resources/dicts/events/lifegen_events/murder.json

```diff
@@ -196,11 +196,11 @@
         "v_c's state of mind has slowly started to deteriorate, causing them to lash out in their confusion. As you try to calm v_c down, they demean you, saying that a youngster such as yourself doesn't know anything. You prove v_c wrong by murdering them while the whole clan slept late that night, a smile gracing your face as you whisper to them that you at least know how to kill."
     ],
     "murder warrior kitten": [
-        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, with deathberry juice dripping from their mouth.",
-        "You offer to take v_c onto a grand and exciting adventure out of the nursery. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind as their startled cries call out your name from behind. You loop back to camp, alone, and no one asks where you've been. v_c is later found dead in the territory.",
+        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, already having gone to StarClan.",
+        "You offer to take v_c onto a grand and exciting adventure out of the nursery. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind. You loop back to camp, alone, and no one asks where you've been. v_c is later found dead in the territory.",
```

Comment:
this can be changed to match the previous one

"You offer to take v_c onto a grand and exciting adventure out of the nursery. You help v_c sneak out, and the two of you run freely through the territory. When you're far enough from camp, you suddenly speed up, leaving v_c behind. You loop back to camp, alone, and no one asks where you've been."